codegen/version-reporter/pkg/reader/goconst.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2023 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reader
+
+import (
+	"fmt"
+	"go/types"
+	"strconv"
+	"strings"
+
+	"golang.org/x/tools/go/packages"
+)
+
+func ReadGoConstantFromPackage(dir string, pkgToSearch string, constName string) (string, error) {
+	config := &packages.Config{
+		Dir:        dir,
+		BuildFlags: []string{"-tags=ee"},
+		Mode:       packages.NeedTypes | packages.NeedTypesInfo,
+	}
+
+	pkgs, err := packages.Load(config, pkgToSearch)
+	if err != nil {
+		return "", err
+	}
+
+	sourcePkg := pkgs[0]
```

Comment:
Shouldn't we log something/error if the list contains more than 1 item?

Reply:
I'm loading a single package, so I don't really expect more than one item and I would think checking for anything else is..... overcautious :D